steps/monitoring-apps-logs/1.0.1/step.yml

```diff
@@ -0,0 +1,49 @@
+title: Mobile apps logs
+summary: |
+  Check if your Android app kept any logs in your release build
+description: "Check if an Android application contain any logs in the release app\nIf
+  yes, this step will fail and generate a failure where you can see the count \n\n###
+  Troubleshooting\n\n- Be aware, you have to launch this step after [apps-decompiler](https://github.com/imranMnts/bitrise-step-apps-decompiler)
+  to have informations from your APK\n"
```

Comment:
Why trailing newline?

Reply:
I don't know
I don't have it in my local repository, may be added during `bitrise share`?

<img width="1206" alt="Screenshot 2021-04-06 at 14 56 00" src="https://user-images.githubusercontent.com/26274254/113714112-5b1ae380-96e8-11eb-9225-2d359d67b84e.png">